avm/res/net-app/net-app-account/capacity-pool/main.bicep

```diff
@@ -91,14 +94,36 @@ module capacityPool_volumes 'volume/main.bicep' = [
       name: volume.name
       location: location
       serviceLevel: serviceLevel
+      networkFeatures: contains(volume, 'networkFeatures') ? volume.networkFeatures : networkFeatures
       creationToken: contains(volume, 'creationToken') ? volume.creationToken : volume.name
       usageThreshold: volume.usageThreshold
       protocolTypes: contains(volume, 'protocolTypes') ? volume.protocolTypes : []
       subnetResourceId: volume.subnetResourceId
       exportPolicyRules: contains(volume, 'exportPolicyRules') ? volume.exportPolicyRules : []
+      zones: contains(volume, 'zones') ? volume.zones : []
       roleAssignments: contains(volume, 'roleAssignments') ? volume.roleAssignments : []
-      networkFeatures: volume.?networkFeatures
-      zones: volume.?zones
```

Comment:
These should not be removed. This syntax automatically pull the default from the referenced volume module.
Remove your lines 97 and 103 and leave these.